Source/Core/DolphinQt/Settings.h

```diff
@@ -120,6 +120,9 @@ class Settings final : public QObject
   bool IsUSBKeyboardConnected() const;
   void SetUSBKeyboardConnected(bool connected);
 
+  void SetIsFrameStepping(bool is_stepping);
```

Comment:
Since this isn't triggered when doing a single frame step, which is the most normal kind of frame step, it would be good to name this something clearer like `SetIsContinuouslyFrameStepping`. Same for the new member variable.